packages/plugin-tanstack-query/package.json

```diff
@@ -28,11 +28,13 @@
 		"lint": "eslint . --config ../../.eslintrc.json",
 		"lint:fix": "eslint . --config ../../.eslintrc.json --fix"
 	},
+	"dependencies": {
+		"graphql-request": "^6.1.0 || ^7.1.2"
```

Comment:
- Not entirely sure why we need this at all, but the previous 7.1.2 conflicts with the one used internally by graphql-codegen.
- I _think_ this is a dependency (since its just a small implementation detail), but could use confirmation.

Reply:
> Not entirely sure why we need this at all?

TRK does not directly support `graphQL` requests directly, it supports promises. `grqphql-request` is required to make promises for query and mutations.

>I think this is a dependency 
True.